jwks/client.go

```diff
@@ -98,7 +98,7 @@ func (c *Client) getRSAPubKeysForIssuer(ctx context.Context, issuerURL string) (
 	return pubKeys, nil
 }
 
-// GetRSAPublicKey gets JWK from JWKS and returns decoded RSA public key. The last one can be used for verifying JWT signature.
+// GetRSAPublicKey gets Key from JWKS and returns decoded RSA public key. The last one can be used for verifying JWT signature.
```

Comment:
Was this change intentional, or was it made accidentally?:)

Reply:
Nope, this was mass replace